packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import { BN } from '@polkadot/util'
+import { Meta, StoryObj } from '@storybook/react'
+import { of } from 'rxjs'
+
+import { joy } from '@/mocks/helpers'
+import { MocksParameters } from '@/mocks/providers'
+
+import { ValidatorList } from './ValidatorList'
+
+type Args = object
+
+export default {
+  title: 'Pages/Validators/ValidatorList',
+  component: ValidatorList,
+  parameters: {
+    mocks: (): MocksParameters => {
+      return {
+        chain: {
+          derive: {
+            staking: {
+              erasRewards: [
+                { era: new BN(688), eraReward: new BN(1234560000) },
```

Comment:
Looks like the story is working but please improve the mock readability a bit.

Here there is no need to wrap the numbers in `BN` because the decorator takes care of that automatically. Also please use the `joy()` function for currency value.

For example here:
```suggestion
                { era: 688, eraReward: joy('0.123456') },
```